recipe/CMakeLists.patch

```diff
@@ -1,56 +1,29 @@
---- CMakeLists.txt	2016-06-16 12:16:15.000000000 -0400
-+++ CMakeLists_New.txt	2017-03-05 01:11:29.000000000 -0500
-@@ -104,11 +104,20 @@
-   
-   set(ARMA_OS macos)
-   
--  set(ARMA_USE_LAPACK true)
--  set(ARMA_USE_BLAS   true)
-+#  set(ARMA_USE_LAPACK true)
-+#  set(ARMA_USE_BLAS   true)
+--- CMakeLists.txt	2016-06-16 18:16:30.000000000 +0200
++++ CMakeLists_new.txt	2017-06-11 13:33:26.000000000 +0200
+@@ -148,9 +148,23 @@
+   set(ARMA_USE_LAPACK true)
+   set(ARMA_USE_BLAS   true)
    
 -  set(ARMA_LIBS ${ARMA_LIBS} "-framework Accelerate")  # or "-framework accelerate" ?
 -  message(STATUS "MacOS X detected. Added '-framework Accelerate' to compiler flags")
-+#  set(ARMA_LIBS ${ARMA_LIBS} "-framework Accelerate")  # or "-framework accelerate" ?
-+#  message(STATUS "MacOS X detected. Added '-framework Accelerate' to compiler flags")
+-  
++  include(ARMA_FindOpenBLAS)
```

Comment:
For consistency with the linux case, should this also have the following up here?:
```  include(ARMA_FindLAPACK)```

If you look at the build logs on TravisCI and CircleCI for ARMA_LIBS, you will see a link to `liblapack.so` on linux, but no corresponding one to `liblapack.dylib` on OS X.  I think adding this would cause it to appear, but I doubt it is really needed because I believe the LAPACK symbols are in `libopenblas` itself and the `liblapack` library is just a symlink to `libopenblas`.